### PR TITLE
Fix jar library loading problem

### PR DESCRIPTION
**Pull request by BFH-ktt1**
This PR addresses #1000

### CODE REVIEW

**Comment by MarcinOrlowski** [comment]
This one fully resolves #1000?

File: src/main/java/com/cburch/logisim/util/LibraryUtil.java [hunk 1]
```diff
@@ -55,7 +54,7 @@ public static String getName(Class cls) {
         iaeEx.printStackTrace();
       }
     }
-
-    throw new NoSuchElementException("Missing " + ID_FIELD_NAME + " static const field for " + cls);
+    /* for backward compatibility we return the class name */
```
Comment (MarcinOrlowski) [inline src/main/java/com/cburch/logisim/util/LibraryUtil.java:57]: Can we have oneliner comments using `// FOO` instead of `/* FOO */`?

**Review by MarcinOrlowski — CHANGES_REQUESTED** [review]

File: src/main/java/com/cburch/logisim/util/LibraryUtil.java (hunk 1 above)
Comment (BFH-ktt1) [reply src/main/java/com/cburch/logisim/util/LibraryUtil.java]: Done

**Comment by BFH-ktt1** [comment]
@MarcinOrlowski , not sure for all libraries, but the jar provided in #1000 loads and works, further testing is required

**Review by MarcinOrlowski — APPROVED** [review]

**Pull request merged**